src/ssh/cockpitsshrelay.c

```diff
@@ -458,7 +459,15 @@ set_knownhosts_file (CockpitSshData *data,
         }
     }
 
-  /* TODO: Check more sources of known hosts here if !host_known */
+  /* check ~/.ssh/known_hosts, unless we are running as a system user ($HOME == "/"); this is
```

Comment:
Okay. Can you please give me a pointer where you would like it? There is no file like `*cockpitrelay*` nor a class like `CockpitRelay`, and this already is in cockpitsshrelay.